File: src/java/eu/sig/training/ch03/Flag.java
package eu.sig.training.ch03;

import java.awt.Color;
import java.util.Arrays;
import java.util.List;

public class Flag {
    private final List<Color> colors;

    public Flag(Color... colors) {
        this.colors = Arrays.asList(colors);
    }

    public List<Color> getColors() {
        return colors;
    }
}

File: src/csharp/eu/sig/training/ch03/Flag.cs
using System.Drawing;
using System.Collections.Generic;

namespace eu.sig.training.ch03
{

    public class Flag
    {
        public IList<Color> Colors { get; }

        public Flag(params Color[] colors)
        {
            Colors = new List<Color>(colors);
        }
    }
}
